apps/els_lsp/src/els_execute_command_provider.erl

```diff
@@ -105,6 +106,85 @@ execute_command(<<"suggest-spec">>, [
         },
     els_server:send_request(Method, Params),
     [];
+execute_command(<<"add-behaviour-callbacks">>, [
+    #{
+        <<"uri">> := Uri,
+        <<"behaviour">> := Behaviour
+    }
+]) ->
+    {ok, Document} = els_utils:lookup_document(Uri),
+    case els_utils:find_module(binary_to_atom(Behaviour, utf8)) of
+        {error, _} ->
+            [];
+        {ok, BeUri} ->
+            %% Put exported callback functions after -behaviour() or -export()
```

Comment:
As a follow up, we could verify whether some callback functions are already exported and add them to the existing list.